### PR TITLE
Making coverage table more responsive

### PR DESCRIPTION
**Pull request by sirgallifrey**
I made a few modifications on the template to make the table work on mobile devices, here are some screenshots

width: 1920 px
cov1914x958

width: 1200 px
cov1207x800

width: 880px
cov880

width: 692px
cov691x799

width: 320px
cov331x809

### CODE REVIEW

**Comment by Trott** [comment]
Thanks for tackling this! /cc @nodejs/website

**Review by watilde — APPROVED** [review]

**Review by mhdawson — APPROVED** [review]
Seems like a good idea.   LGTM

@CurryKitten can you just confirm that the make coverage target you are working on in core still uses this file (I believe so but just want confirmation)

**Comment by mhdawson** [comment]
I think I've answered my own questions and I  think its no since the running summary is not generated yet, so thats ok.  Will plan to land this soon.

**Comment by mhdawson** [comment]
Landed as 4818a1ccede7658b2c61a5ba258af0bfa6d31777, Will check tomorrow to make sure it looks ok on the browsers that I have.

**Comment by sirgallifrey** [comment]
Awesome! Hope everything turns out ok!

File: coverage/generate-index-html.py [hunk 1]
```diff
@@ -32,18 +32,36 @@
         main { margin-bottom: 2rem; }
         .table-header,
         .table-row {
+          box-sizing: border-box;
           display: flex;
           width: 100%;
           padding: 2px 10px;
         }
-        .table-header { font-weight: bold; }
+        .table-header { font-weight: bold;}
```
Comment (stevemao) [inline coverage/generate-index-html.py:40]: Looks like the space after `;` is needed to be consistent.

Comment (sirgallifrey) [reply coverage/generate-index-html.py:40]: Hi @stevemao I need to make another PR now that it has landed or i can just commit to this PR?

Comment (Trott) [reply coverage/generate-index-html.py:40]: You'll need to make another PR. Honestly, I'm not so sure you need to fix that, although by all means, do so if you want to. (There's another line further down with the same issue, for what it's worth, so probably fix them both.)

**Comment by Trott** [comment]
I am really enjoying browsing the coverage stats on my phone. That is all.

**Comment by mhdawson** [comment]
Ok, live and looks good, closing.  @sirgallifrey thanks !

**Comment by mhdawson** [comment]
Mentioned it here: https://developer.ibm.com/node/2017/02/21/easier-node-js-code-coverage-generation/

**Comment by sirgallifrey** [comment]
Wow! Thank you very much @mhdawson! I'm very honored to be mentioned! 😄
One little detail, though, you wrote **firgallifrey** Instead of **sirgallifrey** 😊

**Pull request closed without merging**